src/components/manage/Blocks/Teaser/utils.js

```diff
@@ -1,44 +0,0 @@
-import { isInternalURL } from '@plone/volto/helpers';
-import config from '@plone/volto/registry';
-
-export function getTeaserImageURL({ href, image, align }) {
```

Comment:
We should mention in the upgrade guide that this is no longer used, in case someone has shadowed it.

Reply:
Done.
Also tagging @stevepiercy for this.
My formatter worked hard on some indentation and other formatting in the upgrade guide. I went through all of it, removed something but everything else looks quite legit. I know it's polluting the PR diff but I actually feel like they are good modifications. Let me know what you both think about it.

Reply:
Removed all extra alterations to this file, left only the added lines